### DISCUSSION TITLE
Request: コメントにアイコンでレスポンスしたい

### DISCUSSION THREAD

**Discussion by aviscaerulea** [growilabs/growi#7735] [Ideas / Requests]
優先度の低い話ですが、Slack などのようにコメントにアイコン（絵文字）だけでレスポンスできると嬉しいです。

Growi を緩いコミュニケーションツールとして使おうとしているので、アイコンだけでレスポンスできると楽だしリアクションも取れるので利用者の敷居が下がって嬉しいなと。

| item     | version |
| ---      | --- |
|OS        |Amazon Linux2|
|GROWI     |5.0.4|
|node.js   |16.15.0|
|npm       |8.5.5|
|yarn      |1.22.18|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

**Comment by yuki-takei** [comment]
このご要望は、開発チームが提案している機能アイデア [**スタンプリアクション（Slack 互換）#11137**](https://github.com/growilabs/growi/discussions/11137) として整理されました。

アップボートやコメントをお待ちしております。
